src/content/docs/mobile-apps/android.mdx

```diff
@@ -40,17 +35,27 @@ The user interface for New Relic's Android app provides functionality similar to
   Depending on your New Relic account, you may need to [further authenticate your user](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/user-settings-authentication).
 </Callout>
 
-## Features [#features]
-
-The New Relic Android app includes data about these features:
+<SideBySide>
+  <Side>
+    ## New Relic capabilities [#features]
```

Comment:
Explore adding links to these specific categories unless you think that'd be too much link fatigue.

Reply:
We originally had lnks but my sense was that thye're not particularly helpful in this case and just add to link fatigue.